week-4/A-objects-intro/README.md

```diff
@@ -1,8 +1,23 @@
-### Goals
-
-- introduce objects as a concept
-- when can we use objects (to store what type of information)
-- the very specific object syntax (the commas after each property, the colon between property name and value, the curly brackets)
-- good practices for naming giving names to properties (not having spaces, what to do when there is a space)
-- the language to use when talking about an objects: object literal, properties, property names, property values
-- very simple exercises to get them used to declaring object literals
+### Objects
+
+When you think about real world objects, they have a bunch of properties that describe how its unique. Your laptop, for example, has a brand (Lenovo/Apple etc.), a screen size (13/15 inch), RAM (8/16GB) etc.
+
+As arrays in JavaScript are useful for describing a collection of things, objects are very useful for grouping together a bunch of characteristics to describe a specific entity.

```

Comment:
"Entity" is a technical word/jargon that I don't think adds anything here

Reply:
I'm not sure that the likening to arrays clarifies the description of an object

Reply:
Yep - good point